corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -58,6 +58,15 @@
             }
         };
 
+        var paginationNotice = function (data) {
+            if (data.aaData.length < data.iTotalRecords) {
+                $('#info-message').html("Showing the current page of data. Switch pages to see more data.");
```

Comment:
:+1: Added template tag for translation. I'll leave copywriting choices to @sheelio.

Good point about the UI. I can see how paging might be less obvious with maps than with charts, but maybe paging controls should be at the top and the bottom for all reports.